### ISSUE TITLE
ModuleNotFoundError: No module named 'core'

### ISSUE THREAD

**Issue by xkfx** [volotat/SD-CN-Animation#91]
Error loading script: base_ui.py
Traceback (most recent call last):
  File "/home/ubuntu/stable-diffusion-webui/modules/scripts.py", line 256, in load_scripts
    script_module = script_loading.load_module(scriptfile.path)
  File "/home/ubuntu/stable-diffusion-webui/modules/script_loading.py", line 11, in load_module
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ubuntu/stable-diffusion-webui/extensions/SD-CN-Animation-main/scripts/base_ui.py", line 30, in <module>
    from core import vid2vid, txt2vid, utils
ModuleNotFoundError: No module named 'core'

**Comment by xkfx** [comment]
I encountered a problem when I tried to restart through ./webui.sh, and the plugin is not showing up on my user interface.

**Comment by volotat** [comment]
Try to restart the webui.

**Comment by xkfx** [comment]
I was able to resolve the issue by renaming 'sd-webui-regional-prompter-main' to 'sd-webui-regional-prompter' using the mv command.  The problem might have arisen due to my unconventional installation method, where I directly cloned the repository under 'stable-diffusion-webui/extensions/'.  However, I would like to point out that I did not encounter this issue when installing other plugins using the same method.  It could be beneficial if the author considered refining the path rules to accommodate for this approach.
